xml/gl.xml

```diff
@@ -44079,6 +44080,11 @@ typedef unsigned int GLhandleARB;
             </require>
         </extension>
         <extension name="GL_ARM_shader_framebuffer_fetch_depth_stencil" supported="gles2"/>
+        <extension name="GL_ARM_texture_unnormalized_coordinates" supported="gles2">
+            <require>
+                <enum name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM"/>
```

Comment:
```suggestion
                <enum name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM" group="SamplerParameterF,SamplerParameterI,GetTextureParameter,TextureParameterName" />
```

Reply:
@Perksey  - Thanks. I'm not very familiar with these group parameters, but it looks correct as far as I can tell. Happy to take the suggestion if you think it's complete?

Reply:
I will leave this open for a little while to sort this out. We are basically leaving the group stuff to outside developers like @Perksey so long as it doesn't affect the headers.

Reply:
Doesn't it make more sense to add `group` attribute on line 10342, instead of 44085?

Reply:
Ah whoops, yep that's wrong. Thanks for picking up on that @SunSerega

Reply:
Resolving this conversation - will take the other suggestion.